x-pack/plugins/maps/public/trigger_actions/visualize_geo_field_action.ts

```diff
@@ -49,8 +50,8 @@ export const visualizeGeoFieldAction = createAction<VisualizeFieldContext>({
     const usageCollection = getUsageCollection();
     usageCollection?.reportUiCounter(
       APP_ID,
-      'visualize_geo_field',
-      context.originatingApp ? context.originatingApp : 'unknownOriginatingApp'
+      METRIC_TYPE.CLICK,
+      `create_maps_vis_${context.originatingApp ? context.originatingApp : 'unknownOriginatingApp'}`
```

Comment:
Note, the previous implementation of this UI counter (since 8.2) used a custom string (`'visualize_geo_field'`) for `counterType` rather than one of the preferred `METRIC_TYPE` constants. While this is allowed, I think it makes more sense to use `METRIC_TYPE.CLICK` and have a more descriptive `eventName`. 

Example of previous telemetry 
``` { appName: 'maps', counterType: 'visualize_geo_field', eventName: 'lens', total: 3 } ```

Example of new telemetry
```{ appName: 'maps', counterType: 'click', eventName: 'create_map_vis_lens', total: 3 } ```